rtcp.c

```diff
@@ -1044,3 +1044,345 @@ int janus_rtcp_nacks(char *packet, int len, GSList *nacks) {
 	rtcp->length = htons(words);
 	return words*4+4;
 }
+
+
+inline void set1(guint8 *data,size_t i,guint8 val)
+{
+	data[i] = val;
+}
+inline void set2(guint8 *data,size_t i,guint32 val)
+{
+	data[i+1] = (guint8)(val);
+	data[i]   = (guint8)(val>>8);
+}
+inline void set3(guint8 *data,size_t i,guint32 val)
+{
+	data[i+2] = (guint8)(val);
+	data[i+1] = (guint8)(val>>8);
+	data[i]   = (guint8)(val>>16);
+}
+inline void set4(guint8 *data,size_t i,guint32 val)
+{
+	data[i+3] = (guint8)(val);
+	data[i+2] = (guint8)(val>>8);
+	data[i+1] = (guint8)(val>>16);
+	data[i]   = (guint8)(val>>24);
+}
+
+enum PacketStatus
```

Comment:
Just a nit again (sorry about that, I know how annoying I am! :smile: ), but please don't use camelcase in the code: something like `janus_rtp_packet` would match the code better, and same for the values (e.g., `not_received`).